### ISSUE TITLE
"Negative line number encountered" warning should not occur for "On Error Goto -1"

### ISSUE THREAD

**Issue by Armin1979** [rubberduck-vba/Rubberduck#5689]
As "On Error Goto -1" is a valid statement, which is the only way to reset a previously done "On Error Goto" statement, to go back to the default error handling behavior. Only after using "On Error Goto -1" it is possible to use a new error handler again.
(I personally needed that to use a new error handler inside an error handler :-) (the first handler tries to solve the problem, which also is possible to cause an error)

Currently I getting the warning "Negative line number encountered. Andrew "ThunderFrame" Jackson would be proud! ...." :-)

edit - for clarification: I know about "On Error Goto 0", but for some reason it seems not to be possible to use another "On Error Goto <label>" error handler after "On Error Goto 0", this only worked with "On Error Goto -1" for me. So that's why I would consider it as a valid statement.
See also: https://stackoverflow.com/questions/14158901/difference-between-on-error-goto-0-and-on-error-goto-1-vba
Here it's also stated that "On Error GoTo -1" makes it possible to use another error trap.

**Comment by MDoerner** [comment]
I guess you are using the latest stable version. This bug has already been closed in PR #5594, which is part of a later prerelease.

This issue is basically a duplicate of #5544.

Should you be on the latest prerelease, please reopen.
